### DISCUSSION TITLE
Support for MacOS Silicon (ARM) platform

### DISCUSSION THREAD

**Discussion by nunoguedelha** [ami-iit/matlab-whole-body-simulator#93] [Q&A]
Ciao @traversaro , I've been struggling to run the simulation from the one-line installer on my MacOS Silicon (M1). I realised that it could never work with the pre-built Mex files, since they are platform dependent[^1] . On Anaconda, the `robotology` packages for Apple Silicon are not available yet. Is there a plan to release those in a near future?

[^1]: As you must know, Mex files actually have a different extension, `mexmaca64`(refer to https://fr.mathworks.com/matlabcentral/answers/1928800-can-i-use-mexmaci64-files-on-the-matlab-r2022b-apple-silicon-platform-open-beta)

**Comment by traversaro** [comment]
There is GitHub Actions support for Apple Silicon, so it is a matter of adapting https://github.com/robotology/robotology-superbuild/blob/master/.github/workflows/generate-conda-packages.yaml to run on Apple Silicon . I opened https://github.com/robotology/robotology-superbuild/issues/1659 to track this.
